apstra/api_iba_probes_test.go

```diff
@@ -69,6 +69,12 @@ func TestIbaProbes(t *testing.T) {
 			t.Logf("Label %s", p.Label)
 			t.Logf("Description %s", p.Description)
 			t.Log(p)
+			t.Log("Get IBA probe state")
+			ps, err := bpClient.GetIbaProbeState(ctx, probeId)
+			if err != nil {
+				t.Fatal(err)
+			}
+			log.Printf("Probe state is %s", ps.State)
```

Comment:
Line 67 needs to have the error checked.

Line 78 ("Delete probe") seems like it should come immediately before deleting the probe vs. before enumerating the stages..